ruby-challenges.rb

```diff
@@ -21,6 +47,27 @@
 album3 = 'Abbey Road'
 # Expected output: 'bby Rd'
 
+# define no_vowel(album1)
+# album1 delete ("aeiouAEUOU")
+
+
+def no_vowel string
+    string.delete "aeiouAEIOU"
+    end
+   p no_vowel album1
+   p no_vowel album2
+   p no_vowel album3
```

Comment:
💯!  Just watch your indentation for your end

Reply:
i will been trying to work on that and will keep going